python/redmonster/_version.py

```diff
@@ -1 +1 @@
-__version__ = 'v1.1.0'
+__version__ = '1.1.1.dev3'
```

Comment:
I agree.  This will be fine for this PR, and I can address consistency when it's tagged.